README.md

```diff
@@ -0,0 +1,34 @@
+# Webots Animation Action
+
+This GitHub action creates a Webots animation of a simulation and publishes it to GitHub pages.
+
+<p align="center">
+  <img src="./assets/cover.png">
```

Comment:
It is a pity we don't see the animation slider on this picture. Could you add it?

Reply:
I think it is more descriptive now. However, adding a slider would be too many details?

Reply:
Ok, let's go ahead with this.